bin/subzim

```diff
@@ -66,37 +66,43 @@ PERMISSIONS="rwixd"
 echo "Setting the default permissions: $PERMISSIONS" 
 fi
 
+# zmprov instructions on subscriber account
 PROFFILE="$(mktemp /tmp/subzim-prof.XXXXXXXX.txt)"
+# Sharer's instructions
 BOXMFILE="$(mktemp /tmp/subzim-boxm.XXXXXXXX.txt)"
+# Subscriber instructions
 BOXSFILE="$(mktemp /tmp/subzim-boxs.XXXXXXXX.txt)"
+# Persona creation instructions
+BOXPFILE="$(mktemp /tmp/subzim-boxp.XXXXXXXX.txt)"
 
 echo "" > "${BOXMFILE}"
 echo "" > "${BOXSFILE}"
+echo "" > "${BOXPFILE}"
 echo "" > "${PROFFILE}"
 
 echo "mfg / account $SUBSCRIBER $PERMISSIONS" >> "${BOXMFILE}"
-echo "mfg /Inbox account $SUBSCRIBER $PERMISSIONS" >> "${BOXMFILE}"
```

Comment:
@maxxer can you please re-add these 4 lines, I know they seem redundant, but Zimbra sometimes does not share all folders when calling only the command with `/` as argument. Having these lines saves us a lot of issues reports on Github, thanks.

Reply:
ok